d2/src/main/java/com/linkedin/d2/balancer/strategies/relative/QuarantineManager.java

```diff
@@ -83,7 +85,7 @@ public class QuarantineManager {
     _clock = clock;
     _updateIntervalMs = updateIntervalMs;
     _relativeLatencyLowThresholdFactor = relativeLatencyLowThresholdFactor;
-    _rateLimitedLogger = new RateLimitedLogger(LOG, RelativeLoadBalancerStrategyFactory.DEFAULT_UPDATE_INTERVAL_MS, clock);
+    _rateLimitedLogger = new RateLimitedLogger(LOG, ERROR_REPORT_PERIOD, clock);
```

Comment:
If we use the `_rateLimitedLogger`. Could we change the log level to `warn` ?

Reply:
it is warn level.

Reply:
I mean should we change following log level to warn
```
_rateLimitedLogger.debug("newLb executor rejected new task for getLoadBalancedServiceProperties. "
```
But it's also LGTM to use debug level